ontology/uco/observable/observable.ttl

```diff
@@ -6150,6 +6451,74 @@ observable:Software
 	sh:targetClass observable:Software ;
 	.
 
+observable:SoftwareBuild
+	a
+		owl:Class ,
+		sh:NodeShape
+		;
+	rdfs:subClassOf observable:Software ;
+	.
+
+observable:SoftwareBuildFacet
+	a
+		owl:Class ,
+		sh:NodeShape
+		;
+	rdfs:subClassOf core:Facet ;
+	sh:property
+		[
+			sh:class observable:BuildUtility ;
+			sh:nodeKind sh:IRI ;
+			sh:path observable:buildUtility ;
+		] ,
+		[
+			sh:class observable:Compiler ;
+			sh:nodeKind sh:IRI ;
+			sh:path observable:compiler ;
+		] ,
+		[
+			sh:class observable:Library ;
+			sh:nodeKind sh:IRI ;
+			sh:path observable:library ;
+		] ,
+		[
+			sh:class observable:Package ;
+			sh:nodeKind sh:IRI ;
+			sh:path observable:package ;
+		] ,
+		[
+			sh:class observable:Script ;
+			sh:maxCount "1"^^xsd:integer ;
+			sh:nodeKind sh:IRI ;
+			sh:path observable:buildScript ;
+		] ,
+		[
+			sh:datatype xsd:dateTime ;
+			sh:maxCount "1"^^xsd:integer ;
+			sh:nodeKind sh:Literal ;
+			sh:path observable:compilationDate ;
+		] ,
+		[
+			sh:datatype xsd:string ;
+			sh:maxCount "1"^^xsd:integer ;
```

Comment:
The migration note on `tool:buildOutputLog` suggests moving that property to `observable:`SoftwareBuildFacet with cardinality 0..*.  But, the `observable:` notes had 0..1.  Which should this be?